launch/sample.launch

```diff
@@ -7,8 +7,8 @@
     <arg name="device" value="$(arg device)" />
     <arg name="rate" value="$(arg rate)" />
     <arg name="sensor_frame_id" value="$(arg sensor_frame_id)" />
-    <arg name="rvizconfig" value="$(arg rvizconfig)" />
```

Comment:
I can't find any web site that talks about this, but my own rule of thumb (and I believe this can be effective global ROS-wide) is to port as many variables as argument, so that once someone wants to include your launch file passing some args, s/he can immediately do so.

So I suggest rolling back this change.